sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_native_command.asciidoc

```diff
@@ -181,6 +179,27 @@ any of these things, the behavior is undefined.
 
 == Example
 
+```c++
+sycl::queue q;
+q.submit([&](sycl::handler &cgh) {
+    sycl::accessor acc{buf, cgh};
+    cgh.ext_oneapi_enqueue_native_command([=](sycl::interop_handle &h) {
+      // Can extract device mem from accessor
+      auto src = IH.get_native_mem<backend::ext_oneapi_level_zero>(acc);
+      auto command_list = h.get_native_queue<sycl::backend::level_zero>();
+
+      // Can enqueue arbitrary backend operations. This could also be a kernel
+      // launch or call to a library that enqueues operations on the stream etc
+      //
+      // Important: Enqueuing a *synchronous* backend operation results in
+      // undefined behavior.
+      zeCommandListAppendMemoryCopy(
+              command_list, dst, src, sz, nullptr, 0, nullptr);
```

Comment:
`dst` and `sz` are not defined anyplace.

Reply:
Have updated